environment.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
 - pycodestyle
 - pylint
 - coverage
+- requests
```

Comment:
I think we need to add `ZipFile` here.  And maybe some other of the newly imported packages to `utils.py` (I'm not sure which of those are already dependencies of other packages in this environment, but I'm pretty sure that `ZipFile` is not).

Reply:
These packages are in Python's standard library and are installed by default. They shouldn't need to be in the `environment.yaml` file.

Reply:
Thanks @hdoupe.